Sriram/Reg.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Registration page</title>
+  </head>
+  <body>
+    <h2><strong>Register</strong></h2>
+
+    <form id="Form">
+      First name:
+      <input type="text" name="firstname" value="First name" required/>
+      Last name:
+      <input type="text" name="lastname" value="Last name" /><br /><br />
```

Comment:
Don't use `value`. Refer this: [HTML <input> placeholder Attribute - w3schools.com](https://www.w3schools.com/tags/att_input_placeholder.asp)

Reply:
Also add autocomplete

Reply:
@Nuthi-Sriram Still needs fixing.